parm/field_table.RRFS_sas

```diff
@@ -63,3 +63,23 @@
            "longname",     "subgrid scale turbulent kinetic energy"
            "units",        "m2/s2"
        "profile_type", "fixed", "surface_value=0.0" /
+# prognotsitc sigmab tracer
+ "TRACER", "atmos_mod", "sigmab"
+           "longname",     "sigma fraction"
+           "units",        "fraction"
+       "profile_type", "fixed", "surface_value=0.0" /
+# prognostic smoke mixing ratio tracer
+  "TRACER", "atmos_mod", "smoke"
+            "longname",     "smoke mixing ratio"
+            "units",        "ug/kg"
+       "profile_type", "fixed", "surface_value=1.e-12" /
+# prognostic dust mixing ratio tracer
+            "TRACER", "atmos_mod", "dust"
+            "longname",     "dust mixing ratio"
+            "units",        "ug/kg"
+       "profile_type", "fixed", "surface_value=1.e-12" /
+# prognostic coarsepm mixing ratio tracer
+  "TRACER", "atmos_mod", "coarsepm"
+            "longname",     "coarsepm mixing ratio"
+            "units",        "ug/kg"
+       "profile_type", "fixed", "surface_value=1.e-12" /
```

Comment:
Not sure about `sigmab`, but I believe the rest of these fields are for smoke/dust only

Reply:
Yes, "sigmab" needs to stay, it's a new variable. As to the others - this field_table.RRFS_sas was taken in the same way it was in RRFSv1, which was being developed to include smoke/dust. There are  flags related to smoke/dust options in the `namelist gfs_physics_nml` in FV3.input.yml for RRFS_sas and FV3_HRRR_gf:

rrfs_sd : false
rrfs_smoke_debug : false

They are turned off at the moment. My understanding that they could be turned on as well, when smoke/dust option is made available. 
Please correct me if there is more updated info!

Reply:
@mkavulich - please, let me know if anything else needs attention, and what's the best way to proceed.

Reply:
Depending on the resolution of [this discussion](https://github.com/ufs-community/ufs-srweather-app/discussions/1212) (@benkozi is looking into it), I think these smoke/dust fields should be removed. The field_table simply designates tracers for the model, so if these options aren't being used that's just unnecessary extra memory the model is using to carry around 0-arrays.

Reply:
Thank you, @mkavulich 
Removed three tracer variables for smoke and dust from RRFS_sas field_table.